js/modules/loki_message_api.js

```diff
@@ -102,13 +102,13 @@ class LokiMessageAPI {
       ttl
     );
     // Using timestamp as a unique identifier
-    this.sendingSwarmNodes[timestamp] = lokiSnodeAPI.getSwarmNodesForPubKey(
-      pubKey
-    );
-    if (this.sendingSwarmNodes[timestamp].length < numConnections) {
-      const freshNodes = await lokiSnodeAPI.getFreshSwarmNodes(pubKey);
-      await lokiSnodeAPI.updateSwarmNodes(pubKey, freshNodes);
-      this.sendingSwarmNodes[timestamp] = freshNodes;
+    const swarm = await lokiSnodeAPI.getSwarmNodesForPubKey(pubKey);
+    this.sendingData[timestamp] = {
+      swarm,
+      freshList: false,
```

Comment:
`freshList` => `hasFreshList` to emphasis it's a bool?

Reply:
Good call